data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py

```diff
@@ -83,26 +82,29 @@ def extract(self) -> None:
 
     def transform(self) -> None:
         # Rename GEOID10_TRACT to GEOID10 on score to allow merging with Census GeoJSON
-        self.score_usa_df.rename(
-            columns={self.TRACT_SHORT_FIELD: self.GEOID_FIELD_NAME},
-            inplace=True,
-        )
+        # self.score_usa_df.rename(
+        #     columns={self.TRACT_SHORT_FIELD: self.GEOID_FIELD_NAME},
+        #     inplace=True,
+        # )
 
         logger.info("Pruning Census GeoJSON")
         fields = [self.GEOID_FIELD_NAME, self.GEOMETRY_FIELD_NAME]
         self.geojson_usa_df = self.geojson_usa_df[fields]
-
         logger.info("Merging and compressing score CSV with USA GeoJSON")
-        self.geojson_score_usa_high = self.score_usa_df.merge(
-            self.geojson_usa_df, on=self.GEOID_FIELD_NAME, how="left"
+        self.geojson_score_usa_high = (
```

Comment:
I checked only that the shape of the df was consistent, I didn't check the merge. If we decide to go forward with this I will check it in more detail.